plugins/leaderboard/lib/api/filters/KalturaUserScorePropertiesFilter.php

```diff
@@ -0,0 +1,184 @@
+<?php
+/**
+ * @package plugins.leaderboard
+ * @subpackage api.filters
+ * @abstract
+ * @relatedService UserScoreService
+ */
+class KalturaUserScorePropertiesFilter extends KalturaUserScorePropertiesBaseFilter
+{
+	static private $map_between_objects = array
+	(
+		"userIdEqual" => "_eq_puser_id",
+		"userIdIn" => "_in_puser_id",
+		"rankGreaterThanOrEqual" => "_gte_rank",
+		"rankLessThanOrEqual" => "_lte_rank",
+	);
+	
+	static private $order_by_map = array
+	(
+		"+rank" => "+rank",
+		"-rank" => "-rank",
+	);
+	
+	/**
+	 * @var int
+	 */
+	public $virtualEventId;
+	
+	/**
+	 * @var int
+	 */
+	public $leaderboardId;
+	
+	/**
+	 * @var string
+	 */
+	public $userIdEqual;
+	
+	/**
+	 * @var string
+	 */
+	public $userIdIn;
+	
+	/**
+	 * @var int
+	 */
+	public $ranksAboveUser;
```

Comment:
it doesn't correlate with rankGreaterThanOrEqual rankLessThanOrEqual
above it sounds like you filter on explicit rank and here it relative to a user